include/RED4ext/Scripting/Natives/Generated/anim/AnimVariableBool.hpp

```diff
@@ -18,7 +18,7 @@ struct AnimVariableBool : anim::AnimVariable
     static constexpr const char* ALIAS = NAME;
 
     bool value; // 38
-    bool default_; // 39
+    bool default_; // 39 -- default
```

Comment:
Did you add `-- default` in the dumper?

Reply:
No, it was generated. I didn't touch files of `style` commit before dumping.

Reply:
Well, I'll check to be sure.

Reply:
I confirm, I did *not* add `--default` in the dumper.

Reply:
Strange, not sure why it appears now. Maybe there was some change since the last dump.

Reply:
I think it is related to the commit changing `default` to `default_`. Maybe the generated dump was not run? And in this case the dump adds the missing comment about the real name?